openapi.yaml

```diff
@@ -1375,7 +1376,7 @@ paths:
               example:
                 stac_version: 1.0.0
                 stac_extensions:
-                  - https://stac-extensions.github.io/datacube/v2.0.0/schema.json
+                  - https://stac-extensions.github.io/datacube/v2.2.0/schema.json
```

Comment:
This link does not work yet.

Reply:
Yes, the extension is also waiting for review and release.